packages/create-sitecore-jss/src/templates/nextjs/scripts/generate-config.ts

```diff
@@ -10,11 +10,11 @@ import { JssConfig, jssConfigFactory } from './config';
 */
 
 const defaultConfig: JssConfig = {
```

Comment:
Could we make this more dynamic - iterate through `defaultConfig` keys and set it using `constantCase` module?